src/script/components/InputBar/InputBar.test.tsx

```diff
@@ -108,6 +121,45 @@ describe('InputBar', () => {
     });
   });
 
+  it('typing request is sent if the typing indicator mode is enabled and user is typing', async () => {
+    const props = getDefaultProps();
+    const {container} = render(withTheme(<InputBar {...props} />));
+    const textArea = container.querySelector('textarea[data-uie-name="input-message"]');
```

Comment:
@arjita-mitra the rule of thumb should be: try to use the `container` as little as possible. 

The `container` is a last resort solution when everything else fails. You should first try, in order:
- `getByText` (this is the closest to what the user is going to do)
- `getByTestId` (this is slightly further than the user action, but still controlled since you put the id in there)

in the case of this code, this would look like this

```suggestion
    const {getByTestId} = render(withTheme(<InputBar {...props} />));
    const textArea = getByTestId('input-message');
```